pyproject.toml

```diff
@@ -116,6 +116,7 @@ openai = "^1.31.1"
 seal-rookery = "^2.2.3"
 types-pytz = "^2024.1.0.20240417"
 juriscraper = "^2.6.15"
+django-debug-toolbar = "^4.4.6"
```

Comment:
This should be a dev dep, though, right?

Reply:
You're right, thanks for noticing it. Fixed.